src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -45,7 +45,7 @@ public Game joinGame(@PathVariable("gameId") String gameId, @RequestBody Player
     }
 
     @PostMapping(value = "/{id}/nextQuestion")
-    public Question nextQuestion(@PathVariable("id") String gameId){
+    public Question nextQuestion(@PathVariable("id") String gameId) throws Exception {
```

Comment:
related to respective handling of null pointer exception change

Reply:
See comments above.